pkg/storage/bucket/client.go

```diff
@@ -116,7 +117,7 @@ func NewClient(ctx context.Context, cfg Config, name string, logger log.Logger,
 		return nil, err
 	}
 
-	client = objstore.NewTracingBucket(bucketWithMetrics(client, name, reg))
+	client = t.WrapWithTraces(bucketWithMetrics(client, name, reg))
```

Comment:
NewTracingBucket is now [removed from latest versions](https://github.com/thanos-io/objstore/pull/61) in favour of WrapWithTraces. Need to be sure I'm using the correct package too.

Reply:
nit: we could use `opentracing` as is for the pkg name to make it readable

Reply:
makes sense.. done 👍